From 1bcb43b0efd18f316422942eb64069ffd7e11ce0 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Fri, 18 Jan 2019 11:41:14 +0000
Subject: [PATCH] MANTA-3921 manta-init leaves determining missing images as an
 exercise for the reader Reviewed by: Rui <rui@joyent.com> Approved by: Rui
 <rui@joyent.com>

---
 cmd/manta-init.js | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/cmd/manta-init.js b/cmd/manta-init.js
index 87355c8..7aa6bef 100755
--- a/cmd/manta-init.js
+++ b/cmd/manta-init.js
@@ -7,7 +7,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 /*
@@ -222,8 +222,10 @@ function findLatestImage(service, cb) {
 	    image_name, version_substr, service);
 
 	var onSearchFinish = function (err, image) {
-		if (err)
+		if (err) {
+			log.error(err);
 			return (cb(err));
+		}
 
 		log.info({ image: image }, 'found image %s for %s',
 		    image_name, service);
-- 
2.21.0

